auth-server/test/resolvers/mutation/requestResetPassword.spec.ts

```diff
@@ -40,70 +36,15 @@ describe('requestResetPassword mutation', () => {
 			.query()
 			.where({ id: signupResult.user.id })
 			.del();
-
-		await PasswordResetToken
-			.query()
-			.where({ user_id: signupResult.user.id })
-			.del();
 	});
 
 	it('should be able to request a password reset', async () => {
-
 		const res = await requestResetPassword(null, { email });
 
-		const passwordResetToken = await PasswordResetToken
-			.query()
-			.where({ user_id: signupResult.user.id, valid: true });
-
-		token = passwordResetToken[0].token;
-
-		expect(passwordResetToken.length).to.eq(1);
-		expect(passwordResetToken[0].token).to.not.be.empty;
-		expect(passwordResetToken[0].valid).to.be.true;
 		expect(res.message).to.eq(messages.RESET_PASSWORD_RETURN_MESSAGE);
 	});
 
-	it('should not be able to reset password with a short password', async () => {
-
-		try {
-			await resetPassword(null, { token, newPassword: 'short' });
-		} catch (error) {
-			expect(error).to.exist;
-			expect(error).to.be.an.instanceof(UserInputError);
-			expect(error.message).to.eq(messages.PASSWORD_LENGTH_ERROR);
-		}
-	});
-
-	it('should not be able to reset password with a wrong token', async () => {
-
-		try {
-			await resetPassword(null, { token: uuid(), newPassword });
-		} catch (error) {
-			expect(error).to.exist;
-			expect(error).to.be.an.instanceof(AuthenticationError);
-			expect(error.message).to.eq(messages.PASSWORD_RESET_TOKEN_NOT_FOUND);
-		}
-	});
-
-	it('should be able to reset password with a valid token', async () => {
-
-		const res = await resetPassword(null, { token, newPassword });
-
-		expect(res.message).to.eq(messages.PASSWORD_RESET_SUCCESSFUL);
-	});
-
-	it('should not be able to change password with an expired token', async () => {
-
-		try {
-			await resetPassword(null, { token, newPassword });
-		} catch (error) {
-			expect(error).to.exist;
-			expect(error).to.be.an.instanceof(AuthenticationError);
-			expect(error.message).to.eq(messages.PASSWORD_RESET_TOKEN_INVALID);
-		}
-	});
-
-	it('should not be able to change password with an invalid email', async () => {
+	it('should not be able to request reset password with an invalid email', async () => {
```

Comment:
Let's move this test to resetPassword.

Reply:
This file is testing requestResetPassword not resetPassword mutation. Shouldn't each mutation has sperate tests?

Reply:
That would be better, but as of now, things are all mixed, `resetPassword` has a test called "should be able to request a password reset with token set in redis" which should then rather be in `requetsResetPassword`.

Reply:
done